Updated software citations in documentation (#1157)

itsabhianant opened the pull request:
Fixes #1157

itsabhianant commented:
I have updated the software citations as per Issue #1157 . Please do let me know if any changes need to be done. 

Best,
Abhishek

astrojuanlu commented on docs/source/bibliography.md:
```diff
@@ -52,4 +50,4 @@ IDEs and browsers for my whole academic life.
   community Python package for astronomy.\" *Astronomy & Astrophysics*
   558 (2013): A33.
-  [DOI:10.1051/0004-6361/201322068](http://dx.doi.org/10.1051/0004-6361/201322068)
+  [DOI:10.3847/1538-3881/aabc4f](https://iopscience.iop.org/article/10.3847/1538-3881/aabc4f)
```
The DOI is correct, but the title of the paper changed.

itsabhianant replied:
I can't get where the title is on the website. Is this the title?

```
The Astropy Project: Building an Open-science Project and Status of the v2.0 Core Package*
```

itsabhianant replied:
Or I have to give this title


The Astropy Collaboration, A. M. Price-Whelan, B. M. Sipőcz, H. M. Günther, P. L. Lim, S. M. Crawford, S. Conseil, D. L. Shupe, M. W. Craig, N. Dencheva "The Astropy Project: Building an Open-science Project and Status of the v2.0 Core Package"  *The Astropy Collaboration et al  AJ 156 123*(2018)

astrojuanlu replied:
The second one

astrojuanlu reviewed (changes requested):
Thanks! I requested 1 change

itsabhianant commented:
Hi @astrojuanlu,

I have made the changes as per your instruction.

astrojuanlu commented:
Thanks. There are two merge commits and I'd prefer not to merge them. If you could rebase, squash everything into one commit, and force push, I'd appreciate it .

itsabhianant commented:
Should I close this PR and make a new PR with  a single commit?

DhruvSondhi commented:
If you could rebase & squash that would be better than opening a new PR altogether

itsabhianant commented:
Hi @astrojuanlu & @DhruvSondhi,

I rebased and squashed the commits. Hope now this is good to merge but please let me know if any more changes need to be made.

Best, 
Abhishek

astrojuanlu commented:
Looks like the rebase was not complete :) But the squash was. Won't bother you more with this. Thanks!
